man1/skopeo.1

```diff
@@ -6,121 +6,158 @@
+.TP
+\fBdocker://\fP\fIdocker-reference\fP
+An image in a registry implementing the “Docker Registry HTTP API V2”.
+By default, uses the authorization state in \fB~/.docker/config.json\fP,
+which is set e.g. using \fB(docker login)\fP.
+.TP
+\fBoci:\fP\fIpath\fP\fB:\fP\fItag\fP
+An image \fItag\fP in a directory compliant with “Open Container Image Image Layout Specification”
+at \fIpath\fP.
+
+
 .SH COMMANDS
+.\" ========== skopeo copy ==========
 .TP
-.B copy
+\fBskopeo copy\fP [\fB--sign-by=\fP\fIkey-ID\fP] \fIsource-image\fP \fIdestination-image\fP
 Copy an image (manifest, filesystem layers, signatures) from one location to another.
-.sp
-.B source-location
+.I source-image
 and
-.B destination-location
-can be \fB\%docker://\fRdocker-reference, \fBdir:\fRlocal-path, or \fBatomic:\fRimagestream-name\fB:\fRtag .
-.sp
-\fB\-\-sign\-by=\fRkey-id
-Add a signature by the specified key ID for image name corresponding to \fBdestination-location\fR.
+.I destination-image
+use the “image name” format described above .
+
+With \fB--sign-by=\fP\fIkey-ID\fP,
+add a signature using that key ID for an image name corresponding to \fBdestination-image\fR.
+
 Existing signatures, if any, are preserved as well.
+
+
+.\" ========== skopeo delete ==========
 .TP
-.B delete
-Mark an image for deletion.  You must then run docker registry garabage collection to recover the disk space. E.g.,
+\fBskopeo delete\fP \fIimage-name\fP
+Mark image \fIimage-name\fP for deletion.  You must then run docker registry garabage collection to recover the disk space. E.g.,
 .sp
 \fBdocker exec -it registry bin/registry \\
 .br
 garbage-collect /etc/docker/registry/config.yml\fR
 .sp 2
 Additionally, the registry must allow deletions by setting \fB\%REGISTRY_STORAGE_DELETE_ENABLED=true\fR
 for the registry daemon.
+
+
+.\" ========== skopeo inspect ==========
 .TP
-.B inspect
-Return low-level information on images in a registry
-.sp
-.B image-name
-name of image to retrieve information about
-.br
+\fBskopeo inspect\fP [\fB--raw\fP] \fIimage-name\fP
+Return low-level information about \fIimage-name\fP in a registry.
+
 .B "--raw"
-output raw manifest, default is to format in JSON
+output raw manifest, default is to format in JSON.
+
+
+.\" ========== skopeo layers ==========
 .TP
-.B layers
-Get image layers
-.sp
-.B image-name
-name of the image to retrieve layers
+\fBskopeo layers\fP \fIimage-name\fP
+Get layers of \fIimage-name\fP.
+
+
+.\" ========== skopeo standalone-sign ==========
 .TP
-.B standalone-sign
+\fBskopeo standalone-sign\fP \fImanifest\fP \fIdocker-reference\fP \fIkey-fingerprint\fP \%\fB--output\fP|\fB-o\fP \fIsignature\fP
 Create a signature using local files.
-This is primarily a debugging tool and should not be part of your normal operational workflow.
-.sp
-.B manifest
-path to file containing manifest of image
+This is primarily a debugging tool, or useful for special cases,
+and usually should not be a part of your normal operational workflow;
```

Comment:
I thought we were going to remove this message.  Lets not say it is just a debugging tool.